02_programming_fundamentals/03_algorithms_day_1/.eslintrc.js

```diff
@@ -8,7 +8,10 @@ module.exports = {
     },
     "extends": "eslint:recommended",
     "rules": {
-        "indent": ["error", 2], // 2 spaces indentation
+        "indent": ["error", 2, { // 2 spaces indentation
+          "SwitchCase": 1, // indentation inside switch
+          "ObjectExpression": 1, // indentation inside objects
```

Comment:
Oh 😲 only one for switch and objects?

Reply:
Not, it's just the way of activating the option because eslint is cool and all but the configuration is still way behind its time.